app/src/main/java/com/example/tasky/common/presentation/SimpleButton.kt

```diff
@@ -0,0 +1,47 @@
+package com.example.tasky.common.presentation
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun SimpleButton(
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    enabled: Boolean = true,
+    buttonName: String
```

Comment:
The mandatory arguments should always come at the start of a function, otherwise you can't call it without using named parameters (for composables you want to do that anyways, but it's usually a convention)